app/controllers/post_images_controller.rb

```diff
@@ -0,0 +1,35 @@
+class PostImagesController < ApplicationController
+  before_action :set_post_image, only: [:show]
+
+  PAR_PAGE = 10
+
+  def index
+    @post_images = PostImage.page(params[:page]).per(PAR_PAGE)
+  end
+
+  def show
+  end
+
+  def new
+    @post_image = PostImage.new
+  end
+
+  def create
+    @post_image = PostImage.new(post_image_params)
+    @post_image.user = current_user
+    if @post_image.save
+      redirect_to @post_image, notice: 'Image was successfully created.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.